ci-operator/step-registry/openstack/provision/etcd-on-local-disk/openstack-provision-etcd-on-local-disk-commands.sh

```diff
@@ -150,6 +150,7 @@ spec:
           Where=/var/lib/etcd
           Type=xfs
           Options=defaults,prjquota
+          TimeoutSec=120s
```

Comment:
Can we also seize this opportunity to update the ignition version to the one documented in the product docs? The docs for 4.15, 4.16 and 4.17 all uses ignition version 3.4.0. We should be using the same (we're using 3.2.0, see [line 120](https://github.com/openshift/release/pull/58846/files#diff-3a8fcf6c8250a5783267e1e3a02a100ec735ec59119e203701bfff775fd2800aR120)).

Reply:
Done!